CMakeLists.txt

```diff
@@ -692,6 +692,11 @@ if(HPX_WITH_PARCEL_COALESCING)
   hpx_add_config_define(HPX_HAVE_PARCEL_COALESCING)
 endif()
 
+# Developer tools
+hpx_option(HPX_WITH_VIM_YCM STRING
+  "Generate HPX completion file for VIM YouCompleteMe plugin and save in specified location"
+  "" ADVANCED)
```

Comment:
By default the configuration file could be placed in source directory, YCM looks for every directory above the opened file.